### PR TITLE
Updated scripts to point at new pathes

### PR DESCRIPTION
**Pull request by dhawkes**
Signed-off-by: Dylan Hawkes <dthawkes@google.com>

### CODE REVIEW

File: json_spec/output/dut_info.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
         }
     },
     "required": [
-        "id"
+        "dutInfoId"
```
Comment (mimir-d) [inline json_spec/output/dut_info.json:42]: while i get the problem here, this should not be in this PR; id like to avoid clobbering unrelated code changes, esp stuff that touches the spec

**Review by mimir-d — CHANGES_REQUESTED** [review]
lgtm; just need to extract that spec change

File: json_spec/output/dut_info.json (hunk 1 above)
Comment (dhawkes) [reply json_spec/output/dut_info.json:42]: This is related. I turned on the CI and fixed a bug in the spec. This is not a change in the spec, this is an error. It's requiring a field that doesn't exist. This PR can't be submitted without this change. I'm happy to update the description for this PR to reflect the change better, but creating a second PR for a one line code change is a bunch of unecessary work that accomplishes nothing IMO.

Comment (mimir-d) [reply json_spec/output/dut_info.json:42]: ok, valid argument; i'll approve

**Review by mimir-d — APPROVED** [review]
